hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/s3/multipart/TestS3InitiateMultipartUploadRequest.java

```diff
@@ -69,9 +70,13 @@ public void testValidateAndUpdateCache() throws Exception {
         modifiedRequest.getInitiateMultiPartUploadRequest()
             .getKeyArgs().getMultipartUploadID());
 
-    Assert.assertNotNull(omMetadataManager
+    OmKeyInfo openMPUKeyInfo = omMetadataManager
         .getOpenKeyTable(s3InitiateMultipartUploadRequest.getBucketLayout())
-        .get(multipartKey));
+        .get(multipartKey);
+    Assert.assertNotNull(openMPUKeyInfo);
+    Assert.assertNotNull(openMPUKeyInfo.getLatestVersionLocations());
+    Assert.assertTrue(openMPUKeyInfo.getLatestVersionLocations()
+        .isMultipartKey());
```

Comment:
Thanks for the patch @ivandika3. 
nit: Could you also replace the actual value in assertions at L88 and L93 with `openMPUKeyInfo.getModificationTime()` and `openMPUKeyInfo.getCreationTime()` just to make it cleaner.
Otherwise, LGTM.

Reply:
@SaketaChalamchala Thank you for the review. I have updated the related code.

Reply:
LGTM